examples/esp32/xively_demo/main/user_data.c

```diff
@@ -0,0 +1,361 @@
+#include <stdlib.h>
```

Comment:
copyight? or is this stock ESP32 code?

Reply:
This is all custom. Based on the original ST-Nucleo implementation. I'll make sure those demos have the copyright headers too.